python/GangaDirac/Lib/Files/DiracFile.py

```diff
@@ -463,7 +463,7 @@ def location(self):
                     LFNs.append(this_url)
             return LFNs
 
-    def accessURL(self, thisSE=''):
+    def accessURL(self, thisSE='', protocol='xroot'):
```

Comment:
Actually, the more I think about it the less I like the idea of setting the default. I think it will just cause problems so I'll leave it blank unless anyone thinks the xroot default is necessary.